exonum/src/node/consensus.rs

```diff
@@ -31,9 +32,9 @@ where
     #[cfg_attr(feature = "flame_profile", flame)]
     pub fn handle_consensus(&mut self, msg: ConsensusMessage) {
         // Ignore messages from previous and future height
-        if msg.height() < self.state.height() || msg.height() > self.state.height() + 1 {
+        if msg.height() < self.state.height() || msg.height() > self.state.height().next() {
             warn!(
-                "Received consensus message from other height: msg.height={}, self.height={}",
+                "Received consensus message from other height: msg.height={:?}, self.height={:?}",
```

Comment:
can you implement `Display` for new types, and print it as number, there no reason to debug print in log